LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
- [ ] Kindly fix the MIT license link, it is currently broken

Reply:
Thanks. The MIT license link is actually working...I double-checked it